communitiesuk/funding-service-pre-award-stores
# Fspt 177 apply submit to assess

### Change description
https://mhclgdigital.atlassian.net/browse/FSPT-177
This PR stops application-store from writing to the import queue for assessment. Instead, it calls assessment-store directly to copy the data into the assessment tables. Covered in this PR:
- Updated `submit_application` to call assessment store functions directly to derive application values and insert the row.
- Moved the `send_submit_notification` code into a separate function for easier readability
- Unit tests for the new submit route and functions
- Updated error handling code so that:
    - Submit will not fail for the user if the notification fails to send, or if we can't derive application values

Out of scope:
- Removing the code to read from the queue in assessment. This de-risks the change when it goes into prod and means no data will be lost if someone submits using the old method as we deploy the change.



### How to test
Pull this branch.
Create and submit an application
You should see it in assessment as normal.
No changes should be visible to the user


### Screenshots of UI changes (if applicable)
N/A
